src/RPC/HTTP/Request.php

```diff
@@ -202,7 +202,7 @@ public function getHeader( $name )
 	 */
 	public function getHeaders()
 	{
-		if( is_null( $this->headers == null ) )
+		if( is_null( $this->headers = null ) )
```

Comment:
I think this line should be if( is_null( $this->headers ) ) and the make the protected $headers = null instead of = array.